webcompat/webhooks/helpers.py

```diff
@@ -12,6 +12,8 @@
 from webcompat import app
 from webcompat.helpers import proxy_request
 
+MEDIA_PATH = '/tmp/media_issues.txt'
```

Comment:
yes this is a placeholder for the prototype. 
And it doesn't mean it has to be a flat file it could be a sqlite db too.